src/assets/GLSLAsset.js

```diff
@@ -10,8 +10,15 @@ class GLSLAsset extends Asset {
     this.type = 'js';
   }
 
+  async installParserDependencies() {
+    await localRequire('glslify-deps', this.name);
+    await localRequire('glslify-bundle', this.name);
+    await localRequire('glsl-token-properties', this.name);
```

Comment:
Apparently this is even a bug in the current master branch if looking at the test results

Reply:
would be cool if this were possible to do in one call rather than 4 separate installs. should be faster that way.